src/pages/docs/packaging-applications/build-servers/codefresh-pipelines.md

```diff
@@ -161,6 +161,39 @@ Octopus Deploy steps and examples are available from the [Codefresh Marketplace]
 
 Each step includes one or two examples to help with setting up a workflow. Basic examples include only required arguments, and complex examples include both required and optional arguments.
 
+
+## Deploy a release
+To deploy a release, use the **octopusdeploy/deploy-release** step. Provide details for your Octopus instance, and the project and release you want to deploy:
+```yaml
+deploy-release:
+  type: octopusdeploy/deploy-release
+  arguments:
+    OCTOPUS_API_KEY: '${{OCTOPUS_API_KEY}}'
+    OCTOPUS_URL: '${{OCTOPUS_URL}}'
+    OCTOPUS_SPACE: "Default"
+    PROJECT: "Project Name"
+    RELEASE_NUMBER: "0.0.1"
+    ENVIRONMENTS:
+      - "Development"
+```
+Additionally, you can provide optional arguments to specify guided failure mode or variables:
+```yaml
+deploy-release:
+  type: octopusdeploy/deploy-release
+  arguments:
+    OCTOPUS_API_KEY: '${{OCTOPUS_API_KEY}}'
+    OCTOPUS_URL: '${{OCTOPUS_URL}}'
+    OCTOPUS_SPACE: "Default"
+    PROJECT: "Project Name"
+    RELEASE_NUMBER: "0.0.1"
+    VARIABLES:
+      - "Greeting:Hello"
+    USE_GUIDED_FAILURE: "false"
+    ENVIRONMENTS:
```

Comment:
ENVIRONMENTS after RELEASE_NUMBER would make it easier to compare with the above example